flair/nn/model.py

```diff
@@ -233,6 +233,14 @@ def print_model_card(self):
                 "trained or was trained with Flair version < 0.9.1)"
             )
 
+    @property
+    def supports_smaller_training_vocab(self) -> bool:
+        # the smaller training vocab expects classification tasks, otherwise it won't work.
+        return False
```

Comment:
If I understand correctly, this is True for all models that inherit from Classifier, and False otherwise. If so, could this property be removed from the interface and replaced with an isinstance check in the Trainer?

Reply:
> If I understand correctly, this is True for all models that inherit from Classifier, and False otherwise.

Actually I forgot to add an implementation for the `TextRegressor` and will add that soon. Then this statement would be false.
I would say the correct rule is "Everything that is not generative, is true"